scripts/spin.ps1

```diff
@@ -127,7 +127,11 @@ if ($CodeCoverage) {
 $SpinQuic = $null
 if ($IsWindows) {
     $SpinQuic = Join-Path $RootArtifactDir "spinquic.exe"
-} elseif ($IsLinux -or $IsMacOS) {
+} elseif ($IsLinux) {
+    $SpinQuic = Join-Path $RootArtifactDir "spinquic"
+    $SpinQuicRootDir = Split-Path $SpinQuic -Parent
+    $env:LD_LIBRARY_PATH = "${SpinQuicRootDir}:$env:LD_LIBRARY_PATH"
```

Comment:
Why is this needed only for docker builds?

Reply:
This is still a mystery to me. When I first ran into `libmsquic.so.2: cannot open shared object file` in quicinterop and spinquic, the pipeline was run against the commit, [e079de8](https://github.com/microsoft/msquic/pull/3358/commits/e079de803036340357909911bb103978e09c9a7a), that doesn't even touch the build jobs that quicinterop and spinquic depend on. In other words, making irrelevant build tasks use container somewhat changed the behavior of quicinterop and spinquic, which I don't quite understand.

Reply:
And btw, we don't set rpath for quicinterop and spinquic. So, they are not supposed to find shared libs from the origin folder. I don't know why they even worked before my PR.

Reply:
Maybe because everything is in the same place exactly for build and test machines before?

Reply:
I honest don't know.